libs/search-widget/src/Widget.svelte

```diff
@@ -78,7 +77,7 @@
 
     loadFonts();
     // Load CSS variables (must be done after the CDN was set) and custom styles
```

Comment:
the comment is not accurate anymore, is it?

Reply:
good point